docs/apache-airflow/ui.rst

```diff
@@ -160,6 +160,22 @@ DAG over many runs.
 
 ------------
 
+Landing Times
+.............
+Airflow landing times are calculated from the task's scheduled time to 
+the time the task finishes, either with success or another state (see 
+:ref:`concepts:task-instances`).  When subtracting the task duration 
+(above) from the landing time you will find the adoption time.  If the 
+adoption time exceeds the 
+:ref:`task adoption timeout <config:celery__task_adoption_timeout>` 
+then the task will be terminated and, if applicable, rescheduled.
```

Comment:
what is the explnation for other executors?

Reply:
Great question @eladkal .  We know this is an important factor for the Celery executor, but I do not think there is an equivalent for other executors.  As such, should I pull this out as a note instead, or perhaps move that part of the comment out of the UI section and into the Celery config section that it references?

Reply:
Yeah, this area is a real hodgepodge. I think landing times could be and are relevant to other executors (there is surely some delay between scheduling and the task actually running). But it's pronounced in Celery because of the queuing nature. But Celery isn't the only queuing type executor we'll likely have in the future or that people will build as a third party (think AWS Batch). So perhaps it makes more sense to move the logic for those metrics from `celery_executor.py` to `base_executor.py` at some point in time, and then the docs as well.

But for the moment, my 2c is that this doc contribution is useful to at least get down on paper somewhere and we can optimize the location in the future. I wouldn't block this PR for that.

Reply:
The UI part of the docs comes before the executors part. I'm not sure if it's wise to talk about executor specific before users are aware of what it is. Our docs design to be read as a flow from start to end (and many users on-board to Airflow in that manner)

WDYT?

Reply:
For now I've moved the celery-specific information to the config page entry and linked that back to the UI entry.